Chorus.LinqToXrm.Async/XrmAsyncFetchService.cs

```diff
@@ -57,31 +57,10 @@ public string GetEntityLogicalName(string element)
 
         private async Task Auth()
         {
-            // Turn off spitting out token acquisition logs.
-            LoggerCallbackHandler.UseDefaultLogging = false;
             var ap = await AuthenticationParameters.CreateFromResourceUrlAsync(new Uri($"{_url}{ApiPath}"));
-            var authContext = new AuthenticationContext(@"https://login.microsoftonline.com/common", false);
-
-            AuthenticationResult authResult;
-            if (_clientSecret == null)
-            {
-                authResult = await authContext.AcquireTokenAsync(
-                    ap.Resource,
-                    _clientId,
-                    new Uri(_redirectUrl),
-                    // Workaround for bug https://github.com/MicrosoftDocs/azure-docs/issues/8227
-                    (IPlatformParameters)Activator.CreateInstance(typeof(PlatformParameters), PromptBehavior.Auto)
-                );
-            }
-            else
-            {
-                authResult =
-                    await authContext.AcquireTokenAsync(ap.Resource, new ClientCredential(_clientId, _clientSecret));
-            }
-
-            _client.DefaultRequestHeaders.Authorization =
-                new AuthenticationHeaderValue(authResult.AccessTokenType, authResult.AccessToken);
-
+            var authContext = new AuthenticationContext(ap.Authority, false);
```

Comment:
```suggestion
            var authContext = new AuthenticationContext(@"https://login.microsoftonline.com/common", false);
```
We had a problem in the past that ap.Authority gave a wrong url.